### DISCUSSION TITLE
Latest tag on container images

### DISCUSSION THREAD

**Discussion by scotthenley** [dbt-labs/dbt-snowflake#530] [Q&A]
Why is the "latest" tag on the dbt-snowflake [images](https://github.com/dbt-labs/dbt-snowflake/pkgs/container/dbt-snowflake) still pointing to 1.3 when 1.4 has been out for a few months? We use the latest tag to pull for some of our processes. Thanks!

**Comment by nevdelap** [comment]
I considered raising an issue, but since the choices are bug, feature, and regression and this doesn't fall under those, discussion does seem to be the right place and I upvoted it a while ago, so I wonder why has no one responded to this?

**Comment by nevdelap** [comment]
I decided to raise the issue (https://github.com/dbt-labs/dbt-snowflake/issues/564) since Bug says "Report a bug **or an issue** you've found with dbt-snowflake".

**Reply by nevdelap** [reply]
Now fixed.
